ui_src/src/services/valueConvertor.js

```diff
@@ -337,6 +337,9 @@ export const tieredStorageTimeValidator = (value) => {
 };
 
 export const replicasConvertor = (value, stringToNumber) => {
+    if (isCloud()) {
```

Comment:
Why you didn't pass it by param?

Reply:
resolve in this pr:https://github.com/memphisdev/memphis/pull/1091